apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/DocumentEditor.tsx

```diff
@@ -77,12 +78,13 @@ function DocumentEditorContent({
   freeRunsCount,
   refinementEnabled,
 }: Omit<DocumentEditorProps, 'experimentDiff'>) {
-  const { updateDocumentContent, document } = useDocumentValue()
+  const { updateDocumentContent } = useDocumentValue()
   const [mode, setMode] = useState<'preview' | 'chat'>('preview')
   const { metadata } = useMetadata()
   const { commit } = useCurrentCommit()
   const { project } = useCurrentProject()
   const { toggleDocumentation } = useDeployPrompt()
+  const { document } = useCurrentDocument()
```

Comment:
Why this change. What's different from the these 2 `document`?

Reply:
the other hook needlessly exported document instance, i've just removed that export and favoured useCurrentDocument, which is what we use everywhere.